packages/webpack/bin/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+"""The default binaries used in our local builds
+
+Note that in the npm distribution, this file is absent, and will be generated on-the-fly when
+the package is installed (possibly with different attribute values).
+"""
+
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+nodejs_binary(
+    name = "webpack-worker",
+    data = [
+        "//packages/worker:library",
+        "@npm//webpack-cli",
```

Comment:
Yes, this won’t work for them. this is only for testing and in case of usage from the source. For those who use this as a package, they will get the version that points to `webpack-cli` in the same repository as `@bazel/webpack` was installed. 

So,  //packages/webpack/bin:webpack-worker will be replaced via //@bazel/wepback/bin:webpack-worker as we have `bin` entries in package.json.

Reply:
Also, since we peer dependency on `webpack-cli` it will be visible to worker package.

Reply:
I believe we want this binary to live within a macro such that we have one worker per target.

Reply:
This is how rollup works btw. but if you believe that this is not the right way then we could do the macro approach.

Reply:
got it, this whole nodejs_binary is only for local/from-source use

Reply:
Correct.